brokenspoke_analyzer/core/compute.py

```diff
@@ -303,6 +307,13 @@ def conectivity(
     }
     execute_sqlfile_with_substitutions(engine, sql_script, bind_params)
 
+    # Fetch all road_ids and then calculate stress for them individually but in parallel
+    result = dbcore.execute_query_with_result(
+        engine, "select road_id FROM neighborhood_ways"
+    )
+    road_ids = [e for l in result for e in l]
```

Comment:
Just for info here, when you are trying to do something with iterators, and moreover when that seems to be a very common operation, the `itertools` package probably has the answer you need.

In this case I found out that `https://docs.python.org/3/library/itertools.html#itertools.chain` can be used to flatten a list of tuples directly.

Reply:
Thanks for the suggestion, updated to use `itertools`!